### PR TITLE
tikv: fixed some description about `--db` and `compact-cluster` 

### PR DESCRIPTION
**Pull request by YuJuncen**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

This PR have changed the `--db` option in `compact` and `compact-cluster` to `-d`, due to an unexpected omit of `long` opt <del>when migrating to `structopt`</del>, `--db` is unavailable for a long time.

This PR also explicitly announce that `compact-cluster` needs to specify the pd server instead of individual TiKV nodes.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

https://github.com/tikv/tikv/pull/10837
<del>This PR has accidentally removed the `--db` option... I'm not sure whether we need to fix it. But anyway let's change the document to make it safer.</del>
It seems the previous commit didn't enable the long option too... Perhaps the history of the lost long opt is even longer...

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-control.md [hunk 1]
```diff
@@ -335,7 +335,7 @@ middle_key_by_approximate_size:
 
 ### 手动 compact 整个 TiKV 集群的数据
 
-`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样。
+`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别是 compact 的目标要通过 `--pd` 指定 PD 所在的地址来确定，而不是用 `--data-dir` 或者 `--host` 指定的单个 TiKV。
```
Comment (hfxsd) [inline tikv-control.md:338]:
```suggestion
`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别是 `compact` 的目标要通过 `--pd` 指定 PD 所在的地址来确定，而不是用 `--data-dir` 或者 `--host` 指定的单个 TiKV。
```

File: tikv-control.md [hunk 2]
```diff
@@ -324,18 +324,18 @@ middle_key_by_approximate_size:
...
 
     ```shell
-    tikv-ctl --host ip:port compact --db kv
+    tikv-ctl --host ip:port compact -d kv
     ```
 
 ### 手动 compact 整个 TiKV 集群的数据
 
-`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样。
+`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别是 compact 的目标要通过 `--pd` 指定 PD 所在的地址来确定，而不是用 `--data-dir` 或者 `--host` 指定单个 TiKV。
```
Comment (hfxsd) [inline tikv-control.md:338]:
```suggestion
`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别是使用 `compact-cluster` 命令时，compact 的目标要通过 `--pd` 指定 PD 所在的地址来确定，而不是用 `--data-dir` 或者 `--host` 指定单个 TiKV。
```

**Review by LykxSassinator — APPROVED** [review]
LGTM

File: tikv-control.md [hunk 3]
```diff
@@ -324,18 +324,18 @@ middle_key_by_approximate_size:
...
 
     ```shell
-    tikv-ctl --host ip:port compact --db kv
+    tikv-ctl --host ip:port compact -d kv
     ```
 
 ### 手动 compact 整个 TiKV 集群的数据
 
-`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样。
+`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别是使用 `compact-cluster` 命令时，compact 的目标要通过 `--pd` 指定 PD 所在的地址来确定，而不是用 `--data-dir` 或者 `--host` 指定单个 TiKV。
```
Comment (hfxsd) [inline tikv-control.md:338]:
```suggestion
`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别如下：

- 使用 `compact-cluster` 命令时，通过 `--pd` 指定 PD 所在的地址来确定 compact 的目标。
- 使用 `compact` 命令时，通过 `--data-dir` 或者 `--host` 指定单个 TiKV。
```

**Review by BornChanger — APPROVED** [review]

File: tikv-control.md [hunk 4]
```diff
@@ -314,7 +314,7 @@ middle_key_by_approximate_size:
 
 - `--from` 和 `--to` 选项以 escaped raw key 形式指定 compact 的范围。如果没有设置，表示 compact 整个 TiKV。
 - `--region` 选项指定 compact Region 的范围。如果设置，则 `--from` 和 `--to` 选项会被忽略。
-- `--db` 选项可以指定要 compact 的 RocksDB，可选值为 `kv` 和 `raft`。
+- `-d` 选项可以指定要 compact 的 RocksDB，可选值为 `kv` 和 `raft`。
```
Comment (hfxsd) [inline tikv-control.md:317]:
```suggestion
- `-c` 选项指定 column family 名称，默认值为 `default`，可选值为 `default`、`lock` 和 `write`。
- `-d` 选项指定要 compact 的 RocksDB，默认值为 `kv`，可选值为 `kv` 和 `raft`。
```

**Review by hfxsd — APPROVED** [review]

File: tikv-control.md [hunk 5]
```diff
@@ -324,18 +325,21 @@ middle_key_by_approximate_size:
...
-    tikv-ctl --host ip:port compact --db kv
+    tikv-ctl --host ip:port compact -d kv
     ```
 
 ### 手动 compact 整个 TiKV 集群的数据
 
-`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样。
+`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别如下：
+
+- 使用 `compact-cluster` 命令时，通过 `--pd` 指定 PD 所在的地址来确定 compact 的目标。
```
Comment (hfxsd) [inline tikv-control.md:341]:
```suggestion
- 使用 `compact-cluster` 命令时，通过 `--pd` 指定 PD 所在的地址，以便 `tikv-ctl` 可以找到集群中的所有 TiKV 节点作为 compact 目标。
```

File: tikv-control.md [hunk 6]
```diff
@@ -324,18 +325,21 @@ middle_key_by_approximate_size:
...
+    tikv-ctl --host ip:port compact -d kv
     ```
 
 ### 手动 compact 整个 TiKV 集群的数据
 
-`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样。
+`compact-cluster` 命令可以对整个 TiKV 集群进行手动 compact。该命令参数的含义和使用与 `compact` 命令一样，唯一的区别如下：
+
+- 使用 `compact-cluster` 命令时，通过 `--pd` 指定 PD 所在的地址，以便 `tikv-ctl` 可以找到集群中的所有 TiKV 节点作为 compact 目标。
+- 使用 `compact` 命令时，通过 `--data-dir` 或者 `--host` 指定单个 TiKV。
```
Comment (qiancai) [inline tikv-control.md:342]:
```suggestion
- 使用 `compact` 命令时，通过 `--data-dir` 或者 `--host` 指定单个 TiKV 作为 compact 目标。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Pull request merged**